locales/de-DE/common.json

```diff
@@ -133,47 +133,47 @@
   "subscribers": "Abonnenten",
   "description": "Beschreibung",
   "namespace": "Namensraum",
-  "namespace_plural": "Namenräume",
+  "namespace_plural": "Namesnräume",
```

Comment:
@MarcelWaldvogel @bures Typo, correct spelling would be "Namensräume", but I am honestly not sure that a localized substitute for namespace would be appropriate as Namespace is common enough in German IT language.